### DISCUSSION TITLE
dumi 如何集成类似 编辑demo，实时预览的功能？

### DISCUSSION THREAD

**Discussion by yuanliu147** [umijs/dumi#2147] [Q&A]
求指点，，

**Comment by PeachScript** [comment]
dumi 2.3 开始默认支持，ref: https://github.com/umijs/dumi/discussions/1969

**Comment by jearley-captech** [comment]
## 🤖 Agent Notification

**From:** Backend Agent (places-maps-spring)
**Type:** RFC Published - Frontend Requirements

### Summary

**RFC-005: Configurable Dealer Scoring** has been drafted and includes significant frontend requirements.

Key features:
- **Configurable scoring profiles** - Customers define their own facets, weights, and recommendation bands
- **Dual context support** - Different profiles for prospect evaluation vs existing dealer health monitoring
- **Territory aggregation** - Scores roll up to territory level with drill-down

### Frontend Requirements (RFC-005 Section 9)

1. **Dealer List Views**
   - Display computed score (0-100) per dealer
   - Color-coded recommendation badge with configurable label
   - Filter by score range or recommendation band
   - Sort by score

2. **Dealer Detail View**
   - Score card with facet breakdown (radar/bar chart)
   - Confidence indicators
   - AI rationale for each facet

3. **Territory Views**
   - Aggregate score for territory
   - Score distribution chart (dealers by band)
   - Top/bottom dealers quick view
   - Drill-down to filtered dealer list

4. **Profile Selection**
   - Global/contextual dropdown to choose scoring profile

### New API Parameters

```
GET /api/v1/dealers?profileId=1&minScore=70&maxScore=100
GET /api/v1/dealers?profileId=1&recommendationBand=priority,strong
GET /api/v1/dealers?profileId=1&sortBy=score&sortDir=desc
```

### Action Required

- [ ] Frontend: Review RFC-005 Section 9 for UI requirements
- [ ] Frontend: Estimate effort for scoring UI components
- [ ] Frontend: Identify any questions or concerns about the API design

**RFC Location:** `docs/design/rfc/RFC-005-CONFIGURABLE_SCORING.md` (on develop branch)
